tutorials/vpn-interop-guides/using-cloud-vpn-with-aws/index.md

```diff
@@ -0,0 +1,591 @@
+
+![image](cloudvpnguide---0r6dmoki8tdo.png)
+
+![image](cloudvpnguide---1bhztazlmza.png)
+
+## Configuration - GCP gcloud CLI
+
+Cloud VPN can also be configured using the [gcloud command line tool](https://cloud.google.com/sdk/gcloud/).  Command line configuration requires two steps. First, create the Cloud VPN gateway, then create the tunnels used by the gateway. 
+
+### Reserving an External Static IP Address using the gcloud CLI
+
+Reserve an external static IP address in the Google Cloud Platform network and region where the VPN gateway was created. Make a note of the address created for use in future steps.  
+  
+`gcloud compute addresses create vpn-static-ip --project **vpn-testing /** `
+
+```
+  --region us-central1
+
+### Creating the Cloud VPN Gateway
+
+To create a Cloud VPN gateway, enter the following command:
+gcloud compute target-vpn-gateways create gcp-to-aws / 
+  --network default --region us-central1
+```
+
+### Creating Forwarding Rules
+
+To create the three forwarding rules for the project's network forwarded through the gateway, enter the following commands.
+
+NOTE: the GCP console creates these rules automatically.
+```
+gcloud compute forwarding-rules create gcp-to-aws-rule-udp4500 / 
+  --address gcp-static-ip  --ip-protocol UDP --ports 4500 / 
+  --region us-central1 --target-vpn-gateway gcp-to-aws
+
+gcloud compute forwarding-rules create gcp-to-aws-rule-udp500 / 
+  --address gcp-static-ip --ip-protocol UDP --ports 500 / 
+  --region us-central1 --target-vpn-gateway gcp-to-aws
+
+gcloud compute forwarding-rules create gcp-to-aws-rule-esp / 
+  --address gcp-static-ip --ip-protocol ESP / 
+  --region us-central1 --target-vpn-gateway gcp-to-aws
+```
+
+### Creating the VPN Tunnels 
+
+Since AWS requires two VPN tunnels for redundancy,  perform the following step for each tunnel. For tunnel 2, change the peer-address to a second on-premises IP address and the name to another unique name.  
+  
+
+```
+gcloud compute vpn-tunnels create gcp-to-awstunnel1 / 
+  --peer-address on-prem-IP-1 --ike-version 1 / 
+  --shared-secret SharedSecretGeneratedbyAWS / 
+  --target-vpn-gateway gcp-to-aws / 
+  --local-traffic-selector gcp-CIDR / 
+  --remote-traffic-selector on-prem-CIDR
+```
+
+# IPsec VPN Using Cloud Router
+
+## Configuration - AWS
+
+### Creating the VPC Network
+
+Although new AWS accounts all have a default VPC network, for this exercise, create a new VPC network to connect to the Google Cloud Platform VPN gateway.  
+
+![image](cloudvpnguide---dop5lyp939o.png)
+
+### 
+
+The first step is to select an IP subnet topology.  There are options for various combinations of private and public IP addressing, with or without VPN connectivity.  Once selected, the option cannot be changed.  For the test environment, **Select** a **Private Subnet Only VPC with Hardware VPN Access**:
```

Comment:
Don't think Select needs to be bold.

In the screenshot, the option text reads "VPC with a Private Subnet Only and Hardware VPN Access". Not sure if the screenshot is wrong or the text is.